### DISCUSSION TITLE
Help needed for monitoring additionnal data on AXIS cameras

### DISCUSSION THREAD

**Discussion by sher1ne** [zabbix/community-templates#373] [Q&A]
Hi everyone,
I'm currently using a Zabbix template to monitor my AXIS cameras: [https://github.com/zabbix/community-templates/tree/main/Network_Devices/Other/template_ip_cam_axis_m1013](url) The template works well for basic metrics like uptime, ping, and ICMP response time. However, I would like to monitor additional data points on my Axis cameras, but I am unsure where to find the necessary UUIDs or OIDs.
Could anyone guide me on where to find a  list of OIDs for AXIS cameras or how to identify the UUIDs for additional metrics I can monitor? Any resources or documentation would be greatly appreciated.
@abakaldin

**Comment by kabta1** [comment]
As for finding OID's: AXIS is communicating a few traps, but for the interesting v2 status figures snmpwalk and trial and error is the way.
The existing community template does not seem to work well. I will start working on a new one, but I am new to Zabbix so it may take some time. Let me know if you have any inputs or want to test out the new template.
